packages/bot-web-ui/src/pages/dashboard/announcements/announcements.tsx

```diff
@@ -113,6 +123,18 @@ const Announcements = observer(({ is_mobile, is_tablet, handleTabChange }: TAnno
     };
 
     const handleOnConfirm = () => {
+        rudderStackSendAnnouncementActionEvent({
+            announcement_name: selected_announcement?.announcement.main_title,
+            announcement_action: selected_announcement?.announcement.confirm_button_text,
+        });
+        if (selected_announcement?.announcement.confirm_button_text === 'Import strategy') {
+            rudderStackSendOpenEvent({
+                subpage_name: 'bot_builder',
+                subform_source: 'announcements',
+                subform_name: 'load_strategy',
+                load_strategy_tab: 'recent',
+            });
+        }
```

Comment:
@farabi-deriv let's put this thing inside a specific announcement config's `onconfirm`. We should separate the concern of a specific announcement from the main announcement component.